### PR TITLE
catch the exception and retry logic

### PR DESCRIPTION
**Pull request by hgulersen**

### CODE REVIEW

File: target_redshift/db_sync.py [hunk 1]
```diff
@@ -488,66 +492,153 @@ def load_csv(self, s3_key, count, size_bytes, compression=False):
...
+                            self.logger.info(f"Increased size for: {column}")
+
+                        except:
+                            self.logger.info(
+                                (
+                                    f"Failed to increase size of column {column}. "
+                                    "Will continue with other columns."
+                                )
+                            )
+                            continue
```
Comment (TucumetaL) [inline target_redshift/db_sync.py:639]: We can remove this `continue`

File: target_redshift/db_sync.py [hunk 2]
```diff
@@ -488,66 +492,153 @@ def load_csv(self, s3_key, count, size_bytes, compression=False):
...
+                            self.logger.error(
+                                f"Error running columns size increase: {exc}"
+                            )
+                        finally:
+                            self.column_size_increase_attempted = True
+
+                        self.logger.info(
+                            "Trying again after column size increase attempt"
+                        )
+                        load_data()
```
Comment (TucumetaL) [inline target_redshift/db_sync.py:575]: So if it fails twice we'll let it raise the exception, right?

Comment (hgulersen) [reply target_redshift/db_sync.py]: yeap - it was swallowing the exception, thanks for catching!

**Review by Credgate — APPROVED** [review]
LGTM thanks john! Please test in main when you merge :)

**Pull request merged**